tests/integration/kubernetes/kata-deploy-ensure-runtimec-classes-created.bats

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	expected_runtime_classes=2
+}
+
+@test "Test runtimeclasses are being properly created" {
+	echo "FIDENCIO | DEBUG"
+	current_runtime_classes=$(kubectl get runtimeclasses | grep "kata" | wc -l)
+	echo "FIDENCIO | current_runtime_classes: $current_runtime_classes"	
+	echo "FIDENCIO | expected_runtime_classes: $expected_runtime_classes"	
```

Comment:
I think you might have to force the test to fail to get BATS to show the logs? (or alter a setting elsewhere for that for the whole run?)